lib/player.rb

```diff
@@ -0,0 +1,35 @@
+class Player
+  attr_reader :name, :board
+  
+  def initialize(name)
+    @name = name
+    @board = Board.new
```

Comment:
I think, once we start the game a Board should be initialized, maybe not initialized here.